File: level1/이상한_문자_만들기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(string s) {
    string answer = "";
    int index = 0;
    
    for(auto c: s){
        if(c == ' '){
            index = 0; 
            answer = answer + " ";
        }
        else{
            if(index%2 == 1)
                answer += tolower(c); 
            else
                answer += toupper(c); 
            index++;
        }
    }
    
    return answer;
}

File: JavaScript/level1/이상한_문자_만들기.js
function solution(s) {
    var answer = '';
    let turn = true;
    
    for(let i of s){
        if(i === " "){
            turn = true;
            answer += " ";
        }
        else if(turn){
            turn = !turn;
            answer += i.toUpperCase();
        }
        else{
            turn = !turn;
            answer += i.toLowerCase();
        }
    }
    
    return answer;
}